src/main/resources/suppress.xml

```diff
@@ -7,6 +7,7 @@
 <suppressions>
     <suppress checks="JavadocPackage" files="."/>
     <suppress checks="JavadocVariable" files="."/>
+    <suppress checks="JavadocType" files="."/>
```

Comment:
how about putting this files for static code analysis in /build_config, because they relate to building app and not to operation of all